ls/ls.go

```diff
@@ -54,6 +60,34 @@ func (l *LS) lsDotfiles() {
 	}
 }
 
+// listDirRecursively list all subdirectories encountered from the folder
+func (l *LS) listDirRecursively() error {
+
+	err := filepath.Walk(l.Dir,
+
+		func(path string, info os.FileInfo, err error) error {
+
+			if err != nil {
+				return err
+			}
+
+			//func IsHiddenFile(filename string) (bool, error) {
+			//	return filename[0] == '.', nil
+			//}
```

Comment:
Seems we're not using this since it's commented out. Please remove it

Reply:
Very well.